src/styles/_config.scss

```diff
@@ -27,7 +27,7 @@ $breakpoints: (
 
 // Type
 // -------------------------
-@import url('https://fonts.googleapis.com/css?family=Zilla+Slab|Zilla+Slab+Highlight');
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab|Zilla+Slab+Highlight|Open+Sans');
```

Comment:
I assume this has been carefully and intentionally designed, but it's worth mentioning these fonts add up, especially on mobile, slowing down the page load times, FOUT, etc.